covidatlas/index.css

```diff
@@ -144,10 +144,13 @@ sp-dropdown[emphasized] {
 
 .ca-Landing {
   background-color: var(--BackgroundColor);
+  background-image: url('logo-outlined.svg');
+  background-image: url('logo-outlined.svg'), linear-gradient(123deg, rgba(49,53,113,1) 0%, rgba(30,10,67,1) 100%);
```

Comment:
`background-image: url()` won't work in prod :'(

We'll fix this later, but in the meantime, we have to apply these using `style="background-image: url('`${arc.static('logo-outlined.svg')}');"`